public/sw.js

```diff
@@ -1,15 +1,30 @@
 /* eslint-env serviceworker, browser */
 
-importScripts('/sw-toolbox/sw-toolbox.js');
+importScripts('/sw-toolbox/sw-toolbox.js'); /* global toolbox */
 
-self.addEventListener('install', () => {
-  console.log('INSTALL');
-});
+// The offline page itself *and* its dependencies
```

Comment:
This is annoying. Don't see any easy way to generate this. (Annoying that `/.shell/offline` needs to be a real endpoint, too.)